community/kernel/src/main/java/org/neo4j/kernel/impl/factory/CommunityFacadeFactory.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2002-2015 "Neo Technology,"
+ * Network Engine for Objects in Lund AB [http://neotechnology.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Neo4j is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.neo4j.kernel.impl.factory;
+
+import java.util.Map;
+
+import org.neo4j.graphdb.factory.GraphDatabaseSettings;
+
+import static org.neo4j.helpers.collection.Iterables.append;
+import static org.neo4j.helpers.collection.Iterables.toList;
+import static org.neo4j.kernel.GraphDatabaseDependencies.newDependencies;
+
+/**
+ * This facade creates instances of the Community edition of Neo4j.
+ */
+public class CommunityFacadeFactory
+    extends GraphDatabaseFacadeFactory
+{
+    @Override
+    public GraphDatabaseFacade newFacade( Map<String, String> params, Dependencies dependencies, GraphDatabaseFacade
```

Comment:
I find it harder to read code that have line breaks in weird places, such as between type and variable name. Your IDE should be able to be smarter about this,